tools/src/test/scala/scala/scalanative/nir/SigManglingSuite.scala

```diff
@@ -3,7 +3,7 @@ package nir
 
 import org.scalatest._
 
-class SigManglingSuite extends FunSuite {
+class SigManglingSuite extends funsuite.AnyFunSuite {
```

Comment:
In all other places you added an import for that. Could you do the same here, please?

Reply:
Sure, I originally did that and then went back so I must have missed that one.